fluss-common/src/main/java/com/alibaba/fluss/metadata/TableInfo.java

```diff
@@ -79,20 +96,18 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(tablePath, tableDescriptor, tableId, schemaId);
+        return Objects.hash(tablePath, tableDescriptor, tableId, schemaId, createdTime, modifiedTime);
```

Comment:
remove `createdTime, modifiedTime` from `hashCode()` if they are not compared in `equals()`.